docs/data/components/menu/RadioItems.js

```diff
@@ -143,7 +143,7 @@ const Indicator = styled(Menu.RadioItemIndicator)(
   box-sizing: border-box;
   border-radius: 50%;
 
-  &[data-radioitem=checked] {
+  &[data-radio-item-checked] {
```

Comment:
This should likely just be `[data-checked]`/`[data-unchecked]`?

Reply:
Yep